sdk/go.mod

```diff
@@ -53,6 +52,8 @@ require (
 	google.golang.org/protobuf v1.34.1
 )
 
+require github.com/mitchellh/go-testing-interface v1.14.1 // indirect
```

Comment:
Should this be in the same require block as the other ones down below?

Reply:
Yeah I know what you mean, but the `go mod tidy` put it there 🤔

Reply:
If you move it in and go mod tidy, does it stay? I've sometimes had it be weird like that. It's fine either way to be clear, not a blocker :P

Reply:
I've manually tweaked it down and re-ran `go mod tidy` this time it left it alone. Weird.